kyo-core/shared/src/main/scala/kyo/core.scala

```diff
@@ -241,7 +241,7 @@ object core:
             val tag     = s.tag
         end Continue
 
-        implicit inline def fromKyo[T, S](v: Kyo[T, S]): T < S =
-            v.asInstanceOf[T < S]
+        inline def fromKyo[T, S](inline v: Kyo[T, S]): T < S =
```

Comment:
changes this case to a regular method

Reply:
Perhaps we make this an extension method?

Reply:
I think it'd be odd as an extension method:

```scala
(
      new KyoIO[T, S]:
          def apply(v: Unit, s: Safepoint[IOs & S], l: Locals.State) =
                  f
).fromKyo
```

vs 

```scala
fromKyo (
      new KyoIO[T, S]:
          def apply(v: Unit, s: Safepoint[IOs & S], l: Locals.State) =
                  f
)
```